project-base/app/deploy/deploy-project.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash -e
+
+BASE_PATH="$(realpath "$(dirname "$0")/..")"
+CONFIGURATION_TARGET_PATH="${BASE_PATH}/var/deployment/kubernetes"
+BASIC_AUTH_PATH="${BASE_PATH}/deploy/basicHttpAuth"
+DEPLOY_TARGET_PATH="${BASE_PATH}/var/deployment/deploy"
+CI_ENVIRONMENT_SLUG=${CI_ENVIRONMENT_SLUG}
+SENTRY_DSN=${SENTRY_DSN}
+
+function deploy() {
+    DOMAINS=(
+        DOMAIN_HOSTNAME_1
+        DOMAIN_HOSTNAME_2
+    )
+
+    ENABLE_AUTOSCALING=true
+
+    declare -A PARAMETERS=(
+        ["parameters.trusted_proxies[+]"]=10.0.0.0/8
+    )
+
+    declare -A ENVIRONMENT_VARIABLES=(
+        ["APP_SECRET"]=${APP_SECRET}
+        ["DATABASE_HOST"]=${POSTGRES_DATABASE_IP_ADDRESS}
+        ["DATABASE_PORT"]=${POSTGRES_DATABASE_PORT}
+        ["DATABASE_NAME"]=${PROJECT_NAME}
+        ["DATABASE_USER"]=${PROJECT_NAME}
+        ["DATABASE_PASSWORD"]=${POSTGRES_DATABASE_PASSWORD}
+        ["ELASTICSEARCH_HOST"]=${ELASTICSEARCH_URLS}
+        ["ELASTIC_SEARCH_INDEX_PREFIX"]=${PROJECT_NAME}
+        ["REDIS_PREFIX"]=${PROJECT_NAME}
+        ["MAILER_DSN"]=${MAILER_DSN}
+
+        ["GOPAY_IS_PRODUCTION_MODE"]=${GOPAY_IS_PRODUCTION_MODE}
+        ["GOPAY_EN_GOID"]=${GOPAY_EN_GOID}
+        ["GOPAY_EN_CLIENTID"]=${GOPAY_EN_CLIENTID}
+        ["GOPAY_EN_CLIENTSECRET"]=${GOPAY_EN_CLIENTSECRET}
+        ["GOPAY_CS_GOID"]=${GOPAY_CS_GOID}
+        ["GOPAY_CS_CLIENTID"]=${GOPAY_CS_CLIENTID}
+        ["GOPAY_CS_CLIENTSECRET"]=${GOPAY_CS_CLIENTSECRET}
+
+        ["AKENEO_ENABLED"]=${AKENEO_ENABLED}
+        ["AKENEO_BASE_URI"]=${AKENEO_BASE_URI}
+        ["AKENEO_CLIENT_ID"]=${AKENEO_CLIENT_ID}
+        ["AKENEO_SECRET"]=${AKENEO_SECRET}
+        ["AKENEO_USER"]=${AKENEO_USER}
+        ["AKENEO_PASSWORD"]=${AKENEO_PASSWORD}
+
+        ["SSFWCC_BRIDGE_ENABLED"]=${SSFWCC_BRIDGE_ENABLED}
+        ["SSFWCC_BRIDGE_BASE_URI"]=${SSFWCC_BRIDGE_BASE_URI}
+        ["SSFWCC_BRIDGE_USER"]=${SSFWCC_BRIDGE_USER}
+        ["SSFWCC_BRIDGE_PASSWORD"]=${SSFWCC_BRIDGE_PASSWORD}
```

Comment:
We should probably update name of these ENV variables to only SSFW

Reply:
Among other things... cron instance name, whole ssfwcc bridge namespace in components, config in parameters... 
I would make this change separately if you agree.

Reply:
Ok :)